app/models/waste_carriers_engine/registration.rb

```diff
@@ -33,6 +33,8 @@ def can_start_renewal?
       renewable_tier? && renewable_status? && renewable_date?
     end
 
+    alias renewable? can_start_renewal?
```

Comment:
I appreciate having the alias will save some work now. But if `renewable?` is the name we should be using (because its clearer and consistent with WEX) wouldn't it be better to bite the bullet and rename `can_start_renewal?` instead?

N.B. Where in WEX is [renewable](https://github.com/DEFRA/waste-exemptions-engine/search?q=renewable&unscoped_q=renewable). I can't find it?

Reply:
I got confused with the transient registration `renewal?` method 🤦‍♂ 
I have a lot of places right now in the back-office where I have a registration (which can be transit or not) and I need to check if is renewable. I will actually do that in the presenter for now, as looking back here is not such a good idea